bedrock/mozorg/templates/mozorg/about/governance/policies/security/bugs.html

```diff
@@ -12,10 +12,10 @@ <h1 class="title-banner">{{ _('Handling Mozilla Security Bugs') }}</h1>
 
```

Comment:
I believe @dveditz will be redoing that entire page soon; I just wanted to make a small update to it so that it no longer had incorrect information in bold right on the top.  :)